packages/client/wallets/smart-wallet/src/evm/chains.ts

```diff
@@ -0,0 +1,34 @@
+import { BlockchainIncludingTestnet as Blockchain, objectValues, type ObjectValues } from "@crossmint/common-sdk-base";
+
+const SmartWalletTestnet = {
+    BASE_SEPOLIA: Blockchain.BASE_SEPOLIA,
+    POLYGON_AMOY: Blockchain.POLYGON_AMOY,
+    OPTIMISM_SEPOLIA: Blockchain.OPTIMISM_SEPOLIA,
+    ARBITRUM_SEPOLIA: Blockchain.ARBITRUM_SEPOLIA,
+} as const;
+type SmartWalletTestnet = ObjectValues<typeof SmartWalletTestnet>;
+const SMART_WALLET_TESTNETS: readonly SmartWalletChain[] = objectValues(SmartWalletTestnet);
+
+const SmartWalletMainnet = {
+    BASE: Blockchain.BASE,
+    POLYGON: Blockchain.POLYGON,
+    OPTIMISM: Blockchain.OPTIMISM,
+    ARBITRUM: Blockchain.ARBITRUM,
+} as const;
+type SmartWalletMainnet = ObjectValues<typeof SmartWalletMainnet>;
+const SMART_WALLET_MAINNETS: readonly SmartWalletChain[] = objectValues(SmartWalletMainnet);
+
+const SmartWalletChain = {
```

Comment:
we also support mode, and for now story only on testnet

Reply:
we miss the Mode chain in `@crossmint/common-sdk-base`, will add separately